apps/web/app/(app)/environments/[environmentId]/integrations/mattermost/page.tsx

```
@@ -0,0 +1,52 @@
+    getSurveys(params.environmentId),
+    getIntegrationByType(params.environmentId, "slack"),
+    getEnvironment(params.environmentId),
+    getAttributeClasses(params.environmentId),
+  ]);
+
+  if (!environment) {
+    throw new Error("Environment not found");
+  }
+
+  let channelsArray: TIntegrationItem[] = [];
+  if (slackIntegration && slackIntegration.config.key) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)